.dependency-cruiser.js

```diff
@@ -1,3 +1,46 @@
+function oneOf(...options) {
```

Comment:
I think these functions should be declared in a separate file so this file retains more of a "config file" character.

Reply:
I think that it's a good thing to be able to define functions in config files as those allow one to extract common patterns/names and avoid repetition. In my opinion, it is hard to draw a clear line between what is "configuration" and what is not. Therefore I'm for keeping these functions in `.dependency-cruiser.js`. To de-emphasize them I've moved them to the bottom of the file.